core/sdk/src/framework/file_store.rs

```diff
@@ -45,6 +45,7 @@ where
     fn write_to_disk(&self, value: String) {
         match OpenOptions::new()
             .create(true)
+            .truncate(true)
```

Comment:
Again, I think I have fixed it here https://github.com/rdkcentral/Ripple/pull/246/files#diff-3dcf2dc8b69480c731d175ab085710eed7ff120a2585149d4e27c3087c0a8e19R48 :(